Fix authtoken managment command

sazarkin opened the pull request:
## Problem
Command raised an error when creating token because username param is a list.

## Solution
Two options
- Make username param single argument
- Iterate trough username params

I choose to go with second.

sazarkin commented:
I guess I can address it to original contributor of command @andreagrandi

tomchristie commented:
I think I'd probably prefer that we remove `nargs='+'` and make it a single argument.

sazarkin commented:
Nope it does not work (in my env):
```
$ ➜ python manage.py drf_create_token test@localhost
CommandError: Cannot create the Token: user ['test@localhost'] does not exist
```

I've updated the PR, removed nargs.

tomchristie commented:
Good stuff. Thanks for this!
